railties/lib/rails/test_help.rb

```diff
@@ -7,9 +7,6 @@
 abort("Abort testing: Your Rails environment is running in production mode!") if Rails.env.production?
 
 require "active_support/test_case"
-require "action_controller"
-require "action_controller/test_case"
-require "action_dispatch/testing/integration"
```

Comment:
This removal broke our tests at github. We have been including `Rails::Dom::Testing::Assertions` in our `BasicTestCase` class to use dom assertions like `assert_select`. To fix that I had to manually `require "rails-dom-testing"`. I think it is good that we are loading less stuff by default, but I'm wondering whether it is worth or not a CHANGELOG entry.

Reply:
No need for changelog. The fact that `"rails-dom-testing"` was already loaded by `rails/test_help` was never documented as a feature.